### DISCUSSION TITLE
about imtoken's interface to notify api

### DISCUSSION THREAD

**Discussion by Visrotec** [consenlabs/webview#43] [Q&A] [answered]
I want to know if imtoken has an api interface that can notify us that the customer has transferred to the account
for example 
Someone transferred a sum of money to our cold wallet, is there an interface to notify us, and is there an interface to operate the wallet to pay others.
thanks

**Comment by unix** [comment, marked as answer]
The SDK for WebView only provides functionality to interact with the imToken App, and does not provide any functionality that requires user device permissions. (e.g. Notices, Screen Records, Photos, etc.)
imToken App will not request additional permissions for security purposes and will not pass any permissions to the WebView. 

**So the answer is: NO**

## For DApp (If your business contains)

For your business points, you can ask the user to provide an email address on the WebSite and send an email when appropriate. This is business content that is completely **isolated from imToken**, so you can think and built your business like any traditional website.

## For general users

If the user is using the imToken wallet, they will get a push alert when receives the transfer, which is a device-based push and cannot be called by third-party developers.
